actionpack/test/dispatch/request_test.rb

```diff
@@ -1061,7 +1061,7 @@ class RequestParameterFilter < BaseRequestTest
 
       filter_words << 'blah'
       filter_words << lambda { |key, value|
-        value.reverse! if key =~ /bargain/
```

Comment:
> When a develop want to filtered value to be changed to nil, warnings are noisy. But I think the possibility of these use cases are low.

:+1:, I think before this patch someone has to mutate the value because there was no way to get the `filtered` or `original` value back from the block for next filters.